uber/static/deps/jquery-datetextentry/jquery.datetextentry.js

```diff
@@ -96,15 +96,25 @@
             this[name] = input;
         }
 
+        // ,set_field_widths: function() {
+        //     var opt = this.options;
+        //     var available = this.$element.width() - 2;
+        //     var total = opt.field_width_year + opt.field_width_sep + opt.field_width_month +
+        //                 opt.field_width_sep + opt.field_width_day;
+        //     this.input_day.set_width  ( Math.floor(opt.field_width_day   * available / total) );
+        //     this.input_month.set_width( Math.floor(opt.field_width_month * available / total) );
+        //     this.input_year.set_width ( Math.floor(opt.field_width_year  * available / total) );
+        // }
+
+        // Updated by @RobRuana 2017-05-23
+        // Changes field widths to use percents rather than fixed pixels
```

Comment:
Yeah, I considered submitting a PR. A full PR was going to take too much time, so I decided on waiting until after promo codes were finished. I'd probably need to add a flag to the datetextentry module like, `usePercentWidths: true`, and then update the docs, and add tests, etc...